src/client/components/Button/ButtonComponent.styles.css

```diff
@@ -0,0 +1,35 @@
+.button {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-weight: 700;
+  border: 0;
+  border-radius: 3em;
```

Comment:
I think the component from the mockup does not use a  border-radius

Reply:
yes ... i removed it